exeptions.py

```diff
@@ -0,0 +1,13 @@
+"""ALL EXCEPTIONS FOR GAME"""
+
+
+class GameOver(Exception):
+    pass
+
+
+class EnemyDown(Exception):
+    pass
+
+
+class Exit(Exception):
```

Comment:
This one is useless, just remove it since you're not going to use it in any of your modules.